tests/unit/test_graph.c

```diff
@@ -289,11 +289,11 @@ void test_removeNodes() {
 
 	// validate graph creation
 	TEST_ASSERT(Graph_NodeCount(g) == 3);
-	M = Graph_GetRelationMatrix(g, r, false);
+	M = Graph_GetRelationMatrix(g, r);
 	Delta_Matrix_nvals(&nnz, M);
 	TEST_ASSERT(nnz == 3);
 
-	M = Graph_GetAdjacencyMatrix(g, false);
```

Comment:
the ability to get the transpose matrix is not so good as it is not have the original matrix and if you call to a function like set element it will not update the original
the sync is so rare as we moved to sync only if d+ has 10000 changes or d- has 10000 changes andwe don't sync them both